src/zope/app/broken/interfaces.py

```diff
@@ -16,4 +16,6 @@
 
 __docformat__ = "reStructuredText"
 
-from ZODB.interfaces import IBroken # BBB
+from ZODB.interfaces import IBroken
+
+__all__ = ["IBroken"]
```

Comment:
The BBB comment was useful information.  It implies:

- that this is not the current official API location to import IBroken from
- that it might need to be kept here for backwards compatibility concerns (e.g. old Data.fs files might have IBroken pickled as zope.app.interfaces.IBroken)

I am against removing BBB comments.

Reply:
What should happen instead is a shim that uses `zope.deferredimport` with a clear notice when the import will be removed. Here's a simple example: https://github.com/zopefoundation/DocumentTemplate/blob/3.x/src/DocumentTemplate/cDocumentTemplate.py